test/image/mocks/blank-bar-outsidetext_horizontal.json

```diff
@@ -0,0 +1,33 @@
+{
+  "data": [
+    {
+      "type": "bar",
```

Comment:
Thanks for improving test coverage here! Could you combine these horizontal traces in mock into `blank-bar-outsidetext.json` in a separate subplot? That way we won't introduce too many new mocks

Reply:
Done in 1504463.

Reply:
Thansk!

Reply:
<b>Correction</b>: done in https://github.com/plotly/plotly.js/pull/3817/commits/95076408fb63493313b8c3757ebe2966c521ea8a.